erpnext/patches/v14_0/migrate_existing_lead_notes_as_per_the_new_format.py

```diff
@@ -16,7 +16,7 @@ def execute():
 
 		for d in records:
 			if strip_html(cstr(d.notes)).strip():
-				doc = frappe.get_doc(doctype, d.name)
+				doc = frappe.get_doc({"doctype": doctype, "name": d.name})
```

Comment:
FW added a fix to not child table columns when fetching parent doc: https://github.com/frappe/frappe/pull/18032 

This is probably not required now.